src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -1485,6 +1485,25 @@ static bool filter_chunks(const ObjectRepositoryItem* item)
                 }
             }
             return (_filter_flags & (1 << (RideTypeDescriptors[rideType].Category + _numSourceGameItems))) != 0;
+        }
+        case ObjectType::SmallScenery:
```

Comment:
`default`. A tower is silly, especially since the function only works for rides anyway. Heck, do we even **need** a switch if the function only works for rides?